server/core/region_option.go

```diff
@@ -201,9 +201,13 @@ func SetReadQuery(v uint64) RegionCreateOption {
 // SetWrittenQuery sets the write query for the region.
 func SetWrittenQuery(v uint64) RegionCreateOption {
```

Comment:
Is this method should only be used in test?

Reply:
yes

Reply:
Please add a comment.